packages/hooks/package.json

```diff
@@ -20,7 +20,6 @@
   },
   "dependencies": {
     "@manifest-ui/utils": "^0.0.0",
-    "focus-trap": "^6.7.3",
```

Comment:
This library does not play well with react-portals, hence the change to `react-focus-lock`. Removing in favor of using the new `focus-trap` library. I preserved the API from this to keep the use consistent, allow us to swap libraries out if we need to in the future.